server/views/components/template.njk

```diff
@@ -23,10 +26,31 @@
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-152065860-4"></script>
     <script>
       window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
+      function gtag() {
+        dataLayer.push(arguments);
+      }
       gtag('js', new Date());
       gtag('config', 'UA-152065860-4');
     </script>
+    {% if hotJarId %}
+      {# Load HotJar #}
+      <script>
+        (function (h, o, t, j, a, r) {
+          h.hj = h.hj || function () {
+            (h.hj.q = h.hj.q || []).push(arguments)
+          };
+          h._hjSettings = {
+            hjid: '{{hotJarId}}',
+            hjsv: 6
+          };
+          a = o.getElementsByTagName('head')[0];
+          r = o.createElement('script');
+          r.async = 1;
+          r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
+          a.appendChild(r);
+        })(window, document, 'https://static.hotjar.com/c/hotjar-', '.js?sv=');
```

Comment:
Are there some kind of allow lists that this will need to be added to? presumably on the prison side?

Reply:
Good point - getting confirmation on this

Reply:
I've created a card to arrange this work with CTS. This code can be merged as without the configuration in Circle it won't effectively be live